src/bin/cql-stress-cassandra-stress/main.rs

```diff
@@ -1,3 +1,25 @@
-fn main() {
-    todo!("The cassandra-stress frontend is not implemented yet");
+mod settings;
+
+#[macro_use]
+extern crate lazy_static;
+
+use crate::settings::parse_cassandra_stress_args;
+use anyhow::Result;
+use std::env;
+
+use settings::CassandraStressParsingResult;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // Cassandra-stress CLI is case-insensitive.
+    let payload = match parse_cassandra_stress_args(env::args().map(|arg| arg.to_lowercase())) {
```

Comment:
The current design of `parse_cassandra_stress_args` has side effects of printing parse errors to stdout - this could be redesigned to instead return error messages in `Err` and print them here (or in `CassandraStressParsingResult::Special` value). One potential benefit of that are unit tests that check the error messages (but not really sure if they are that useful).